Enlight/SKF.Enlight.csproj

```diff
@@ -1,7 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Version>1.0.6</Version>
```

Comment:
I don's see a need to specify the version in csproj, instead its specified when building as a flag.
Don't want a new commit for each version bump.